src/app/feature/user/pages/user-debug/user-debug.page.ts

```diff
@@ -141,6 +142,17 @@ export class UserDebugPage implements OnInit {
     return { headers, rows };
   }
 
+  public async resetSelectedDynamicData() {
+    const selected = this.dynamicDataSelected();
+    await this.dynamicDataService.resetFlow("data_list", selected);
+    location.reload();
+  }
+
+  public async resetAllDynamicData() {
+    await this.dynamicDataService.resetAll();
+    location.reload();
```

Comment:
nit(non-blocking): resetting all data also probably doesn't need full reload. As this is a debug page I think we can expect the user to manually refresh if something doesn't appear quite right after reset (but I'd expect main use case would be author staying on page and observing a single table which should still be fine)